network/msgp_gen.go

```diff
@@ -0,0 +1,1088 @@
+package network
+
+// Code generated by github.com/algorand/msgp DO NOT EDIT.
+
+import (
+	"github.com/algorand/msgp/msgp"
+)
+
+// The following msgp objects are implemented in this file:
+// disconnectReason
```

Comment:
looks like it's not hurting but do we actually need to generate methods for this type or is it safe to `msgp:ignore` it?

Reply:
No clue, my goal was to get msgp working for the networking package. As others start using msgp in networking, they'll need to make sure the generated files work as intended for them.

Reply:
My guess is not if you are not using it and network didn't run `msgp:generate` prior to your commit. nbd but you can `msgp:ignore` it which will cut down a few tests and remove unnecessary interface implementation